src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -132,6 +136,44 @@ const OrgProfileTab: FC = () => {
     </FlexBox.Child>
   )
 
+  const LeaveOrgButton = () => {
+    const currentUserId = me.id
+    const {users, handleRemoveUser} = useContext(UsersContext)
+
+    const allowSelfRemoval = users.length > 1
+
+    const handleRemove = () => {
+      handleRemoveUser(currentUserId)
+      window.location.href = CLOUD_URL
+    }
+
+    return (
+      <>
+        {allowSelfRemoval && (
+          <FlexBox.Child>
+            <h4>Leave Organization</h4>
+            <p className="org-profile-tab--heading org-profile-tab--deleteHeading">
+              Leave the <b>{org.name}</b> organization.
```

Comment:
thoughts on text under the `Leave Org` title?

Reply:
Sounds good to me.